admin/services/config.service.js

```diff
@@ -59,9 +60,10 @@ const configService = {
       winston.error('Failed to get access arrangements: ' + error.message)
     }
 
-    if (accessArrangementsCodes.includes('ATA')) {
-      checkOptions.audibleSounds = true
-    }
+    accessArrangementsCodes.forEach(code => {
+      if (code === 'ATA') checkOptions.audibleSounds = true
```

Comment:
I'd prefer not testing against magic strings.   I think testing against a constant makes more sense, at least I would know what it meant.

Reply:
Definitely agree - updated to use what George has done recently now that it's in master.